### DISCUSSION TITLE
Release build issue

### DISCUSSION THREAD

**Discussion by AkashMicheal** [infinitered/ignite#1914] [Q&A] [answered]
when we are using infinity red ignite architecture for react native it works excellent in debug mode but when we generate .ipa or .apk files its shows a white screen only Does anyone know what the issue is
![5b75a642-8875-4b17-a9dc-8b6c9aecbec9](https://user-images.githubusercontent.com/71168987/156746780-56eb87d0-f68c-492d-8a91-7ae5918d8750.jpeg)
?

**Comment by jamonholmgren** [comment, marked as answer]
Tracking this in issue https://github.com/infinitered/ignite/issues/1908.
